### ISSUE TITLE
search results page - accessibility - text line length for page description text is too wide

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#104]
accessibility guidelines states that there should be max of 80 char per line (if wider its hard for users with dyslexia and congestive problems to read)

can line width be 150 px less

![search_results_page_-_accessibility_-_text_line_length_for_page_description_text_is_too_wide](https://user-images.githubusercontent.com/2722522/65516507-08e4e980-ded9-11e9-8ecb-ed45c254714d.png)

**Comment by enginehouse** [comment]
fixed
